src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -62,7 +62,7 @@
  * @version $Id$
  * @since 0.9
  */
-@SuppressWarnings("PMD.TooManyMethods")
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.GodClass"})
```

Comment:
@essobedo please add a new puzzle to refactor this class and remove the new suppression above